features/support/pages/appointment_summary_page.rb

```diff
@@ -140,6 +143,10 @@ def fill_in_format_preference(appointment_summary)
     end
   end
 
+  def fill_in_digital_request(appointment_summary)
```

Comment:
You might consider this overkill given this is used once and requires more code than just inlining this method. Although, I appreciate there is an emergent pattern being followed here.